README.md

```diff
@@ -266,18 +266,18 @@ in the following order, with optional plugin provided scripts in bold:
 You can modify the pipeline with the following option:
 
 ```
-$ ./generate_travis_config.py --help
-usage: generate_travis_config.py [-h] [--exclude-docs-test]
-                                 [--exclude-mariadb-test]
-                                 [--exclude-deploy-to-pypi]
-                                 [--exclude-test-bindings]
-                                 [--exclude-deploy-client-to-pypi]
-                                 [--exclude-deploy-client-to-rubygems]
-                                 [--exclude-deploy-daily-client-to-pypi]
-                                 [--exclude-deploy-daily-client-to-rubygems]
-                                 [--pypi-username PYPI_USERNAME]
-                                 [--exclude-check-commit-message]
-                                 plugin_name
+$ ./bootstrap.py --help
```

Comment:
i don't think this helptext is accurate.

Reply:
Would you mind if I just show the help command and remove the output?

Reply:
Nevermind, I've just re-rendered the help text